monitor/seaclouds-data-collector/src/main/java/eu/seaclouds/monitor/seacloudsDc/DcService.java

```diff
@@ -0,0 +1,45 @@
+package eu.seaclouds.monitor.seacloudsDc;
```

Comment:
I think the right package should be `eu.project.seaclouds.monitor.datacollector`